app/templates/index.hbs

```diff
@@ -1,4 +1,8 @@
-<ul class="menu align-center menu--selector">
+{{#orange-bar}}
+  {{#orange-bar-custom-map-study}}{{/orange-bar-custom-map-study}}
+{{/orange-bar}}
```

Comment:
Good. 

You can also leverage folders for component organization. For example:

orange-bar/custom-map-study.hbs would be:

```handlebars
{{#orange-bar}}
   {{orange-bar/custom-map-study}}
{{/orange-bar}}
```

Note that components without blocks can be invoked without `#`.